analysis/run_supercloud_experiments.sh

```diff
@@ -2,22 +2,32 @@
 
 START_SEED=456
 NUM_SEEDS=5
-ENVS=(
-    "cover"
-    "blocks"
-    "painting"
-    "repeated_nextto"
-    "cluttered_table"
-)
-APPROACHES=(
-    "oracle"
-    "nsrt_learning"
-)
+FILE="analysis/submit.py"
 
 for SEED in $(seq $START_SEED $((NUM_SEEDS+START_SEED-1))); do
-    for ENV in ${ENVS[@]}; do
-        for APPROACH in ${APPROACHES[@]}; do
-            python analysis/submit.py --env $ENV --approach $APPROACH --seed $SEED
-        done
-    done
+    # cover
+    python $FILE --env cover --approach oracle --seed $SEED
+    python $FILE --env cover --approach nsrt_learning --seed $SEED
+    python $FILE --env cover --approach grammar_search_invention --seed $SEED
+    python $FILE --env cover --approach grammar_search_invention --seed $SEED --excluded_predicates all
+
+    # blocks
+    python $FILE --env blocks --approach oracle --seed $SEED
+    python $FILE --env blocks --approach nsrt_learning --seed $SEED
+    python $FILE --env blocks --approach grammar_search_invention --seed $SEED
+    python $FILE --env blocks --approach grammar_search_invention --seed $SEED --excluded_predicates all
+
+    # painting
+    python $FILE --env painting --approach oracle --seed $SEED
+    python $FILE --env painting --approach nsrt_learning --seed $SEED
+    python $FILE --env painting --approach grammar_search_invention --seed $SEED --grammar_search_grammar_includes_givens 0 --grammar_search_grammar_includes_foralls 0
```

Comment:
Just checking, will it be problematic that the save config str thing does not take into account commandline overrides?

Reply:
as we usually do, the responsibility is on the user to not override their own results files. if you want to run different experiments and make sure everything is saved out, you'll need to add fields to the save/config str.